05-transform.Rmd

```diff
@@ -955,7 +955,11 @@ plot(polyg, col = "grey", main = "POLYGON")
 par(par_old)
 ```
 
-This process can be also reversed using `st_cast`:
+Transition from mulitpoint to linestring is a common operation used to convert a set of ordered point observations, such as GPS measurements or geotagged media, into a path.
```

Comment:
I think this line should be:

```
Conversion from mulitpoint to linestring is a common operation used to convert a set of ordered point observations, such as GPS measurements or geotagged media, into a path.
```

Transition implies a gradual process with intermediary steps, as in the **transformr** package (which we should mention!) for example: https://twitter.com/thomasp85/status/982223090561179648

Reply:
Feel free to change it;)